test/powershell/engine/ETS/Adapter.Tests.ps1

```diff
@@ -249,6 +255,12 @@ Describe "Adapter Tests" -tags "CI" {
         It "Can use PSWhere as an alias for the Where magic method" {
             $x = 5
             $x.PSWhere{$true} | Should -Be 5
+
+            $p = $null.PSWhere{"I didn't run"}
+            $p.GetType().Name | Should -BeExactly 'Collection`1'
+            $p.Count | Should -BeExactly 0
+
+            ([pscustomobject]@{ Name = 'bar' }).PSWhere({$_.Name}) | ForEach-Object Name | Should -BeExactly 'bar'
```

Comment:
Could you use a more appropriate name instead of `bar`? Once upon a time, we specifically got rid of `foo` and `bar` in tests.

Reply:
'bar' here is a string value, not the name of variable or similar.